homeassistant/components/telegram_bot/__init__.py

```diff
@@ -573,6 +587,12 @@ def _make_row_inline_keyboard(row_keyboard):
             ATTR_REPLYMARKUP: None,
             ATTR_TIMEOUT: None,
             ATTR_MESSAGE_TAG: None,
+            ATTR_SUPPORTS_STREAMING: None,
+            ATTR_DURATION: None,
+            ATTR_WIDTH: None,
+            ATTR_HEIGHT: None,
+            ATTR_THUMB_URL: None,
+            ATTR_THUMB_FILE: None,
```

Comment:
It is possible to make the code more compact and readable/maintainable.

Before the `params` variable, add:
```python
if data is None:
	data = {}
```

Now you can simplify the code:

```suggestion
            ATTR_SUPPORTS_STREAMING: data.get(ATTR_SUPPORTS_STREAMING),
            ATTR_DURATION: data.get(ATTR_DURATION),
            ATTR_WIDTH: data.get(ATTR_WIDTH),
            ATTR_HEIGHT: data.get(ATTR_HEIGHT),
            ATTR_THUMB_URL: data.get(ATTR_THUMB_URL),
            ATTR_THUMB_FILE: data.get(ATTR_THUMB_FILE),
```

Please also adapt it for the other attributes, where appropriate.